### PR TITLE
Validate WebClips urls

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1431
[Problem] WebClips urls are not validated
[Solution] Validate url of each WebClip. If not valid, remove it from WebClips list.

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: examples/mobile/HomeApp/js/app.js [hunk 1]
```diff
@@ -74,6 +74,38 @@ import Storage from "./clipping-storage.js";
...
+	async function validateAppsList() {
+		let app,
+			webClip;
+
+		for (app of appsList) {
+			for (webClip of app.webClipsList) {
+				try {
+					// we wait for the every fetch response so validateAppsList
+					// will return when all job is done, not earlier
+					const response = await fetch(webClip.url + "\\webclip.html");
```
Comment (TomaszLukawskiSam) [inline examples/mobile/HomeApp/js/app.js]: I think you should try to do this simultaneously

Comment (lslachciak) [reply examples/mobile/HomeApp/js/app.js]: I used Promise.allSettled i new approach

**Comment by lslachciak** [comment]
I'll update this PR when https://github.com/Samsung/TAU/pull/1420 merged

**Comment by lslachciak** [comment]
I verified this PR on mobile too using
https://lslachciak.github.io/TAU/examples/1.2/examples/mobile/HomeApp/
Chrome and Samsung Internet
if you access this site you can see alert OK
this means that code didn't crash after Promise.allSettled call

```
responses = await Promise.allSettled(promisesList);
alert("ok");

```

**Review by TomaszLukawskiSam — COMMENTED** [review]
Verified. Looks good to me.

**Review by TomaszLukawskiSam — APPROVED** [review]

**Pull request merged**